pulpcore/cli/common/generic.py

```diff
@@ -807,6 +862,18 @@ def _type_callback(ctx: click.Context, param: click.Parameter, value: t.Optional
     expose_value=False,
 )
 
+repositories_option = resource_option(
+    "--repository",
+    "repositories",
+    context_table=PulpRepositoryContext.TYPE_REGISTRY,
+    multiple=True,
+    href_pattern=PulpRepositoryContext.HREF_PATTERN,
+    help=_(
+        "Repository in the form '[[<plugin>:]<resource_type>:]<name>' or by href."
+        " Can be called multiple times."
+    ),
+)
+
```

Comment:
I don't see this being reused anywhere. Why should we make it generic? (Most other places use an additional capability.)

Reply:
@mdellweg I just think it is generic by nature, choosing multiple repositories just doesn't seem too specific in the context of Pulp imo

Reply:
The resource option is the generic thing here.
When selecting arbitrary repositories, usually we need to inject an additional capabilities. This makes this version just not so generic.